test/functional/cypress/integration/ecs/clusters_list.spec.js

```diff
@@ -51,4 +53,39 @@ describe('Amazon ECS: aws-prod-ecsdemo cluster', () => {
       .get('a:contains("Rollback")')
       .click();
   });
+
+  it('shows stored details view and ECS server group actions1', () => {
+    cy.visit('#/applications/ecsapp/clusters');
+
+    cy.get('.sub-group:contains("aws-prod-ecsdemo")')
+      .find('.server-group:contains("v000")')
+      .find('.cluster-container')
+      .find('.instances')
+      .find('.instance-group')
+      .get('a[title="f8757e00-184d-4288-b535-4124a739e7be"]')
+      .click();
+
+
+    cy.get('.details-panel > .header')
+      .get('instance-details-header')
+      .get('.header-text')
+      .get('h3:contains("5880e2f5d22e485d834d1d38861147aa")');
+
+    cy.get('.details-panel > .content > .collapsible-section').eq(0)
+      .get('.content-body > dl > dd');
+
+    cy.get('.details-panel > .content > .collapsible-section').eq(1)
+      .get('.content-body > dl > dd').eq(0)
+      .get('span:contains("Up")');
```

Comment:
there's a lot of drilling down through DOM elements here in order to get and check the content of this span. This makes the specific UI context we're asserting on hard to pin down and brittle to future changes in the page layout. 

Can we instead add [test IDs](https://github.com/spinnaker/deck/blob/master/test/functional/cypress/integration/ecs/server_group.spec.js#L38-L39) to the relevant spans/DOM elements in the instance detail component and discover them that way?

Reply:
I have changed functional test to have a minimal level of drilling down through the DOM elements. Please take a look at it and let me know.